scripts/teigaku_genzei/make_icl.py

```
@@ -0,0 +1,119 @@
+
+def generate_QA_combinations(row_sr: pd.Series, qa_df: pd.DataFrame, size: int)-> list[tuple[int, list[int]]]:
+    q_list = cast(list[int], json.loads(row_sr["qlist"]))
+    if len(q_list) >= size:
+        comb_triples = itertools.combinations(q_list, size)
+        comb_triples_list = [(cast(int, row_sr.name), list(comb)) for comb in comb_triples]
+        return comb_triples_list
+    elif len(q_list) > 0:
+        comb_triples_list = [(cast(int, row_sr.name), (q_list + [q_list[0]] * (size - len(q_list))))]
+        return comb_triples_list
+    else:
+        return []
```

Pylint (4.0.0)

[refactor] Unnecessary "elif" after "return", remove the leading "el" from "elif"

(R1705)